frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -98,6 +98,11 @@
 
         vm.workerLogs = [];
 
+        // get from backend
+        vm.selectedWorkerResources = [0.5, 1024];
```

Comment:
can we store these in 100s instead of floating point? Like the boto3 API expects? You can show it like this in dropdown ` 256 vCPU (0.25 CPU)` ` 512 vCPU (0.5 CPU)`. We should ensure the API parameters are in the boto3 units

Reply:
Sure